### DISCUSSION TITLE
Request to join as a layer 2 table (Community R&D Table)

### DISCUSSION THREAD

**Discussion by Deathranger14** [Xahau/xahaud#215] [Governance Pitches]
We are a group of indivuals who have been associated with the XRPL/Community for 5+ years and as a team are running a Validator/Xpop Node in support of Xahau. Regardless of if we are accepted in this vote, We plan on supporting the Network in multiple ways from Technical support to PR with an Educational motive to spread knowledge on the existing possibilites of Xahau and help educate others when joining the community, helping them on their learning journey adding to the infrastucture and support. our Validators Key is  nHDNfBFyCyZBKHaeRhNfpJkdgGKWg2Li5p4gUdKyDZRfeyNrQh9a . we have established a website with details o n our table and hope to hear from others on the network in regard to our request to join the governance game as a layer 2 table! The link below has more details on our team! https://xahl.xrpl.community/

**Comment by WietseWind** [comment]
Thank you for your proposal! The very first 🎉  Thanks for "kicking this off".

Speaking for XRPL Labs, but pretty sure the same applies to the other governance members, I think your proposal would need some more information.

There are no hard rules about what's needed, that's the idea: no one dictates what's good or bad. But good and complete ideas will help starting off a conversation. If there was a guideline and everyone would follow the guideline you get similar L2s.

It's actually very interesting if proposals come up with their own view on governance, inclusivity, goals, communication arrangements, etc. Methods don't matter, as long as they are well thought through, by people who can are trusted & have a track record, and things like voting, expansion, goals, audience, etc. etc. are well thought through, and make the entire ecosystem better.

**Reply by Deathranger14** [reply]
Thank you for being the first reply back! It's a learning proccess and i love that we are going through it first. I just updated our website with more information on it but plan on getting some free time this week where i can sit down and generate what i feel to be a nicely done website with proper information and contact refrences. If any other members on the Layer 1 or Layer 2 of Xahau have any questions for us please feel free to reach out to us via the email we put on the website or under our discussion here on GitHub. I look forward to future discussions on the topic and please do not hesitate to reach out if there is specific information you would like to have when considering your Vote for our table to join the Layer 2 governance game! I will post later this week when i have further updated our website.
